### PR TITLE
Resolves stale.yml exempting some PR and Issue labels

### LINKED ISSUE #1381: Stale issue marking not working properly
**Describe the bug**
When the issue is marked as inactive `no activity` label cannot be removed by issue author in any way

image

**Example issues**
#856 #855 #844 #809 #779 #963 #769

**Expected behaviour**
The label should be removed after the author post any message on issue like `still valid`

**Affected workflow**
https://github.com/CircuitVerse/CircuitVerse/blob/master/.github/workflows/stale.yml

**Are you working on this issue? (Yes/No)**
No

### PR DESCRIPTION
**Pull request by ShubhankarKG**
#### Describe the changes you have made in this PR -
Exempted certain labels from being added to stale.

### CODE REVIEW

**Comment by ShubhankarKG** [comment]
Please review @MrBartusek @tachyons

**Comment by ShubhankarKG** [comment]
That is exactly what @Ananthavijay told in the comment reference.

**Comment by MrBartusek** [comment]
> That is exactly what @Ananthavijay told in the comment reference.

He was just wrong, sorry for the inconvenience

**Comment by Ananthavijay** [comment]
Yeah, sorry about that

**Comment by nitin10s** [comment]
We don't need to exempt PRs I think.

**Comment by ShubhankarKG** [comment]
Are the issue labels valid ? In that case I'll remove the PR labels. @nitin10s

File: .github/workflows/stale.yml [hunk 1]
```diff
@@ -17,3 +17,6 @@ jobs:
         stale-pr-message: ' No activity on this PR for a prolonged duration, will be closed if no further activities in next 7 days'
         stale-issue-label: 'no-issue-activity'
         stale-pr-label: 'no-pr-activity'
+        exempt-issue-labels: 'enhancement, Feature, good first issue, help wanted, pending-product-review'
+        exempt-pr-labels: 'enhancement, Feature, help wanted, pending-product-review, Simulator, New_UI'
```
Comment (nitin10s) [inline .github/workflows/stale.yml:21]: Remove this

File: .github/workflows/stale.yml [hunk 2]
```diff
@@ -17,3 +17,6 @@ jobs:
         stale-pr-message: ' No activity on this PR for a prolonged duration, will be closed if no further activities in next 7 days'
         stale-issue-label: 'no-issue-activity'
         stale-pr-label: 'no-pr-activity'
+        exempt-issue-labels: 'enhancement, Feature, good first issue, help wanted, pending-product-review'
```
Comment (nitin10s) [inline .github/workflows/stale.yml:20]: re-check the labels, as I've changed their names, also add bug to the list

Comment (ShubhankarKG) [reply .github/workflows/stale.yml]: Resolved in new commit.

File: .github/workflows/stale.yml (hunk 1 above)
Comment (ShubhankarKG) [reply .github/workflows/stale.yml]: Resolved in new commit

**Comment by ShubhankarKG** [comment]
Please review @nitin10s

**Comment by tachyons** [comment]
Instead of whitelisting all labels, we can create new labels like do_not_close to prevent important issues/Prs being closed. Whitelisting all of them defeat the purpose of having a stale bot

**Comment by ShubhankarKG** [comment]
Even that is possible. But in case, we use a kind of [Issue-Label-Bot](https://github.com/marketplace/issue-label-bot) in future that could automate a bug, feature request or question label, then we'd have to mark them again. Nevertheless, this can be implemented.

**Comment by nitin10s** [comment]
@tachyons I am not sure if we even need the stale bot for Issues? We close them manually if they are not relevant. And if they are, they should never be closed. Right?

**Pull request closed without merging**